### DISCUSSION TITLE
Remove sqflite

### DISCUSSION THREAD

**Discussion by 781flyingdutchman** [781flyingdutchman/background_downloader#187] [Ideas]
The plugin adds substantially to the size of the package, and is only used by those who need SqLite functionality. 

We can move classes like `SqlitePersistentStorage` to the example app, so they can still be used by copying them into the developer application.

Planned for V8

**Comment by 781flyingdutchman** [comment]
In V8, moved all SQLIte related code to a separate package (not yet published at the time of this comment, as it requires V8 to actually be published first!). Add `background_downloader_sql` to your dependencies in pubspec.yaml to get `SqlitePersistentStorage` and SQLite related migration options back.

The reason for this change is that the `sqflite` dependency adds significant size to apps, even if they do not use the SQLite functionality.

**Reply by markst** [reply]
Thanks for this!
